apps/src/code-studio/assets/SpriteUpload.js

```diff
@@ -58,22 +75,75 @@ export default class SpriteUpload extends React.Component {
     });
   };
 
+  handleAvailabilityChange = event => {
+    this.setState({spriteAvailability: event.target.value});
```

Comment:
is there any other state we want to clear/reset when availability changes?

Reply:
I added a category reset here as well to force a user to re-choose a category in case they've gone back and forth. Other than that, I don't think any other state needs to be reset.